nvtabular/io/dataset.py

```diff
@@ -18,9 +18,13 @@
 import random
 import warnings
 
-import cudf
+try:
+    import cudf
+    import dask_cudf
+except ImportError:
+    cudf = None
+    dask_cudf = None
```

Comment:
Not sure if we should wait for "full" cpu support before covering the case that cudf/dask_cudf is not installed...

Reply:
Yeah - I think fully handling the case where there is no cudf/dask_cudf will require changes to almost every module here and is probably out of scope of the 0.4 release.  I think we can wait on this for the 'full' support in 0.5